### PR TITLE
Syndicate Uniform resprite and loadout additon

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
Replaces the ancient bay syndicate uniform with a recolored ironhammer turtleneck. it can also now be found in maint

## Why It's Good For The Game
more character customization and reworks an old and ugly sprite

## Changelog
:cl:
add: added syndicate turtleneck to the loadout section
tweak: unblack list and rename the syndicate turtleneck
tweak: resprites syndicate turtleneck
:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
14-Apr-22 19-42-08

**Comment by SirRichardFrancis** [comment]
5 months old branch with 23 unrelated commits? C'mon, just make a new one.
That sprite looks like an IH uniform with poop colored pants, not like a Syndicate combat turtleneck.
Certainly should not delete the old sprite, while it not compliant with Eris style, still have more character to it than this.
Consider updating old one instead.

**Comment by ValoTheValo** [comment]
> 5 months old branch with 23 unrelated commits? C'mon, just make a new one. That sprite looks like an IH uniform with poop colored pants, not like a Syndicate combat turtleneck. Certainly should not delete the old sprite, while it not compliant with Eris style, still have more character to it than this. Consider updating old one instead.

bro it adds these commits whenever I pull from upstream. and make forks of my master.

**Comment by ValoTheValo** [comment]
do you want me to add the old sprite as unused for archival purposes?

**Comment by SirRichardFrancis** [comment]
> bro it adds these commits whenever I pull from upstream. and make forks of my master.

You're doing it wrong. Just reset your Master, smh.

**Comment by SirRichardFrancis** [comment]
> do you want me to add the old sprite as unused for archival purposes?

I want you to either update it to comply with Eris spriting style (best option) or just leave it alone (that's fine), certainly not removing it from anywhere.

File: code/modules/clothing/under/syndicate.dm [hunk 1]
```diff
@@ -1,9 +1,8 @@
 /obj/item/clothing/under/syndicate
-	name = "tactical turtleneck"
-	desc = "It's some non-descript, slightly suspicious looking, civilian clothing."
+	name = "Syndicate Turtleneck"
+	desc = "An old corporate war era Ironhammer uniform, worn as standard by Syndicate marines. it consists of a black turtleneck and cargo pants. It reminds you of a different time"
```
Comment (SirRichardFrancis) [inline code/modules/clothing/under/syndicate.dm:3]:
```suggestion
	desc = "An old Corporate War era uniform, worn by Syndicate operatives. It consists of a black turtleneck and cargo pants."
```

File: code/modules/client/preference_setup/loadout/lists/uniforms.dm [hunk 2]
```diff
@@ -186,3 +186,7 @@
 /datum/gear/uniform/battledress_german
 	display_name = "battle dress uniform, oberth"
 	path = /obj/item/clothing/under/germansuit
+
+/datum/gear/uniform/syndicate
+	display_name = "Syndicate turtleneck"
+	path = /obj/item/clothing/under/syndicate
```
Comment (SirRichardFrancis) [inline code/modules/client/preference_setup/loadout/lists/uniforms.dm:192]: No newline.

Comment (ValoTheValo) [reply code/modules/client/preference_setup/loadout/lists/uniforms.dm:192]: what?

Comment (SirRichardFrancis) [reply code/modules/client/preference_setup/loadout/lists/uniforms.dm:192]: Check [this](https://lmgtfy.app/?q=why+file+should+end+with+newline).

Comment (ValoTheValo) [reply code/modules/client/preference_setup/loadout/lists/uniforms.dm:192]: > Check [this](https://lmgtfy.app/?q=why+file+should+end+with+newline).

ah i thought you meant i shouldnt newline. allright added one

**Comment by ValoTheValo** [comment]
My sprite sucks ass and people are mad about it. Ill try making a 2nd version

**Pull request closed without merging**